### DISCUSSION TITLE
May 2024 upgrade support

### DISCUSSION THREAD

**Discussion by fergalish** [SoftwareVerde/bitcoin-verde#23] [General]
Title says all - any timeline on support for ABLA May 2024 upgrade? Thanks.

**Comment by joshmg** [comment]
Soon™

In short, we're still ensuring the migration from MySQL to LevelDB is stable before we upgrade to ABLA--our logic is that upgrading to the ABLA doesn't make sense if the node crashes.  That being said, the migration is complete and performs very well!  We're just shoring some things up and we'll move on ABLA.
